main.go

```diff
@@ -242,12 +242,47 @@ func main() {
 			return nil
 		},
 	}
+	
+	znnCliPlasmaGet := &cli.Command{
+		Name: "plasma.get",
+		Usage: "",
+		Action: func(cCtx *cli.Context) error {
+			if cCtx.NArg() != 0 {
+				fmt.Println("Incorrect number of arguments. Expected:")
+				fmt.Println("plasma.get")
+				return nil
+			}
+
+			kp, err := getZnnCliSigner(walletDir, cCtx)
+			if err != nil{
+				fmt.Println("Error getting signer:", err)
+				return err
+			}
+			z, err := connect(url, chainId)
+			if err != nil{
+				fmt.Println("Error connecting to Zenon Network:", err)
+				return err
+			}
+			plasmaInfo, err := z.Embedded.Plasma.Get(kp.Address())
+			if err != nil {
+				fmt.Println("Error getting plasma info:", err)
+				return err
+			}
+			currentPlasma := plasmaInfo.CurrentPlasma
+			maxPlasma := plasmaInfo.MaxPlasma
+			qsrAmount := plasmaInfo.QsrAmount
```

Comment:
[is that QsrAmount a raw amount in bigInt?](https://github.com/zenon-network/go-zenon/blob/master/rpc/api/embedded/plasma.go#L37)

The qsrAmount is big.Int

You can see in the dart znn-cli that it formats it first
https://github.com/zenon-network/znn_cli_dart/blob/master/cli_handler.dart#L281

You can do the same with the formatAmount function available
https://github.com/ignition-pillar/nomctl/blob/master/main.go#L35
ZNN and QSR both have 8 decimals